site/styles/_mixins.scss

```diff
@@ -23,3 +23,17 @@
     z-index: 8;
     @include gradient-x( rgba($left-gradient-bg-color, 1), transparent, 0%, 100%);
 }
+
+.a-links{
```

Comment:
Can you please use the [mixin syntax](https://sass-lang.com/documentation/at-rules/mixin) so there is no doubt what this is for!

Reply:
You will need @include in the places you use it instead of @extend too.

Reply:
Initially I thought to use mixins and includes like the sass examples, but our default styles don't seem to be aligned with that. I mean instead we declare mixins in the _mixins.scss and use @extend to use it.

Reply:
Yeah the existing stuff isn't great... but we should everything properly going forwards!